internal/metrics/metrics.go

```diff
@@ -32,3 +34,21 @@ type Metrics interface {
 	Latency(ctx context.Context, query Query, offset time.Duration, alignReduceType AlignReduce) (float64, error)
 	ErrorRate(ctx context.Context, query Query, offset time.Duration) (float64, error)
 }
+
+// PercentileToAlignReduce takes a percentile value maps it to a AlignReduce
+// value.
+//
+// TODO: once we start supporting any percentile value, this should not be
+// needed.
+func PercentileToAlignReduce(percentile float64) (AlignReduce, error) {
+	switch percentile {
```

Comment:
@broady kinda random but because of this https://play.golang.org/p/3of70nwGNo-
do you recommend we use `if .. else if` with `>=`